### DISCUSSION TITLE
Decorating words when moving editor to next group

### DISCUSSION THREAD

**Discussion by Ramin-RX7** [microsoft/vscode-discussions#493] [Extension Development QnA] [answered]
I am currently working on an extension that uses `activeEditor.setDecorations` on some specific keywords. It works perfectly and I even added `vscode.window.onDidChangeActiveTextEditor` and `vscode.workspace.onDidChangeTextDocument`  so the window will be update on editor changes. Here is the problem:
Imagine I have to windows open. 
1) the file my extension is applied to   2) any other open window
If my active window is `2` and I use the command `workbench.action.moveEditorToNextGroup` (which will cause my custom file be visible in the left side), my decorations are not applied unless I change my active editor to it (like using `workbench.action.quickOpenPreviousRecentlyUsedEditorInGroup`). Is there any update event that will be triggered when I do this?

**Comment by r3m0t** [comment, marked as answer]
That sounds like a bug- but try using `visibleTextEditors` instead.
